aspnet/mvc/views/forms/sample/final/ViewModels/AddressViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace FormsTH.ViewModels
+{
+    public class AddressViewModel
+    {
+        public string AddressLine1 { get; set; }
+    }
+
+    public class RegisterViewModel2
+    {
+        public string Email { get; set; }
+
+        public string Password { get; set; }
```

Comment:
_Never_ demonstrate such a poor practise. This means users will be typing their password in clear text.